plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -220,9 +221,15 @@ func (a *Elasticsearch) Connect() error {
 		Timeout:   time.Duration(a.Timeout),
 	}
 
+	elasticURL, err := url.Parse(a.URLs[0])
+	if err != nil {
+		return err
```

Comment:
Can we please add some unique error text here to ease debugging?
```suggestion
		return fmt.Errorf("parsing URL failed: %v", err)
```